src/SFA.DAS.PensionsRegulatorApi.UnitTests/Application/Queries/GetValidatedOrganisations/Given_Repository_Returns_Results.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using MediatR;
+using NSubstitute;
+using NUnit.Framework;
+using PensionsRegulatorApi.Application.Queries;
+using PensionsRegulatorApi.Data;
+using PensionsRegulatorApi.Domain;
+
+namespace SFA.DAS.PensionsRegulatorApi.UnitTests.Application.Queries.GetValidatedOrganisations
+{
+    [TestFixture]
+    [ExcludeFromCodeCoverage]
+    public class Given_Repository_Returns_Results
+    {
+        private IOrganisationRepository _mockRepository;
+        private IRequestHandler<global::PensionsRegulatorApi.Application.Queries.GetValidatedOrganisations, IEnumerable<Organisation>> _sut;
+        private IEnumerable<Organisation> _repositoryResults;
+
+        public Given_Repository_Returns_Results()
+        {
+            _repositoryResults =
+                new Fixture()
+                    .CreateMany<Organisation>(
+                        new Random()
+                            .Next(1, 15));
+        }
+
+        [SetUp]
+        public void Given()
+        {
+            _mockRepository = Substitute.For<IOrganisationRepository>();
+
+            _mockRepository
+                .GetOrganisationsForPAYEReferenceAndAORN(Arg.Any<string>(), Arg.Any<string>())
```

Comment:
Any should be show with sh1t for setups, can we use fixed values?

Reply:
I have now re-written the query tests in a different style; you may not like this new style, so I may have to go back to GWT and rename :)

Whilst doing this I decided to rename the queries to be more representative as well (specifically 'validate' was no longer being used in the api, so having queries with that name didn't make sense).